src/proj.h

```diff
@@ -313,10 +335,12 @@ double proj_xyz_dist (XYZ a, XYZ b);
 
 /* Set or read error level */
 int  proj_errno (PJ *P);
-int  proj_errno_reset (PJ *P);
 void proj_errno_set (PJ *P, int err);
-/* reduce some mental impedance in the canonical reset/restore use case */
-#define proj_errno_restore(P, err) proj_errno_set ((P), (err))
+int  proj_errno_reset (PJ *P);
+void proj_errno_restore (PJ *P, int err);
+
+char *proj_get_definition (PJ *P);
```

Comment:
To avoid potential cross heap issues on Windows (if proj is compiled with MSVC X and the application integrating it with MSVC Y), we should provide a proj_free() (if that name is still avaialble) that does the free inside the lib.
( with the "old" proj_api.h, this was the pj_get_def() / pj_dalloc() tuple)